src/service/users.js

```diff
@@ -117,10 +117,36 @@ const editProfileImgDoneHandler = async (req, res) => {
   }
 };
 
+const reportHandler = async (req, res) => {
+  try {
+    const { reportedId, type, etcDetail, time } = req.body;
+    const user = await userModel.findOne({ id: req.userId });
+    const creatorId = user._id;
+    console.log(creatorId, reportedId);
+
+    let report = new reportModel({
+      creatorId: creatorId,
+      reportedId: reportedId,
+      type: type,
+      etcDetail: etcDetail,
+      time: time,
+    });
+
+    await report.save();
+    res.send(200).send("User/report : report successful");
```

Comment:
```suggestion
    res.status(200).send("User/report : report successful");
```
이렇게 안하면 계속 crash 하네요